extensions/ui-extension/src/targets/purchase/thank-you/block/render.tsx

```diff
@@ -1,14 +1,29 @@
 import { Community, Newsletter, GuideLinks } from '@/components/post-purchase';
 import { reactExtensionWithSentry } from '@/helpers/extension';
-import { BlockStack } from '@shopify/ui-extensions-react/checkout';
+import { clearIfixitCart } from '@/helpers/iFixitAPI';
+import { useIfixitSettings } from '@/hooks/useIfixitSettings';
+import {
+   BlockStack,
+   useAttributes,
+   useShop,
+} from '@shopify/ui-extensions-react/checkout';
 
 export default reactExtensionWithSentry(
    'purchase.thank-you.block.render',
-   () => (
-      <BlockStack>
-         <Community />
-         <Newsletter />
-         <GuideLinks />
-      </BlockStack>
-   )
+   () => {
+      const attributes = useAttributes();
+      const publicCartid = attributes?.find(
+         (attr) => attr.key === 'publicCartid'
+      )?.value;
```

Comment:
I think this can be expressed more simply with `useAttributeValues`:

```suggestion
      const [publicCartid] = useAttributeValues(['publicCartid']);
```

Reply:
ok i agree, but im a little concerned that it only appears in the [2023-07](https://shopify.dev/docs/api/checkout-ui-extensions/2023-07/react-hooks/attributes/useattributevalues) docs and not the [2023-10](https://shopify.dev/docs/api/checkout-ui-extensions/2023-10/apis/attributes) docs. I could be reading this wrong, but im interpreting this as its not in the current version?

Reply:
It's in [2023-10](https://shopify.dev/docs/api/checkout-ui-extensions/2023-10/apis/attributes#useAttributeValues), I think they reformatted their docs or something in that version. It's also in [2024-01](https://shopify.dev/docs/api/checkout-ui-extensions/2024-01/apis/attributes#useAttributeValues) which is what we're using.